### DISCUSSION TITLE
What about the Tomcat issue?

### DISCUSSION THREAD

**Discussion by cridus** [MarcGiffing/bucket4j-spring-boot-starter#101] [Q&A] [answered]
Hello, I'd really like to use your library but unfortunatley as of now I'm forced to run my app in Tomcat (might even become Tomcat 10 in the future).
Are there news about issue #43?
Not to put pressure on you or anything, but the last post in there is from 2019. Just asking.

Thank you for your work!

**Comment by MarcGiffing** [comment, marked as answer]
I'm sorry. I will not support the standalone Tomcat. I don't have the time to test it.
